pkg/controller/pod/pod_controller.go

```diff
@@ -132,12 +138,20 @@ var _ reconcile.Reconciler = &ReconcilePod{}
 type ReconcilePod struct {
 	client     client.Client
 	scheme     *runtime.Scheme
-	status     *statusmanager.StatusManager
+	status     statusmanager.StatusManager
 	sharedInfo *sharedinfo.SharedInfo
 
 	nsxNcpResources []types.NamespacedName
 }
 
+type ReconcilePodK8s struct {
+	ReconcilePod
```

Comment:
maybe use a pointer here?

Reply:
ReconcilePodK8s was removed in the new patch set.